### PR TITLE
Seed default label data using migration

### PR DESCRIPTION
**Pull request by PeterOu8**
Seeded default labels into table and removed pre-seeded data.

### CODE REVIEW

File: blotztask-api/Data/Seeding/BlotzContextSeed.cs [hunk 1]
```diff
@@ -59,28 +59,15 @@ public static async Task SeedBlotzContextAsync(UserManager<User> userManager, Ro
...
-                },
-                new Label
-                {
-                    Name = "Completed",
-                    Color = "Green",
-                    Description = "Tasks that have been completed"
-                }
-            );
+            await context.Labels.AddRangeAsync();
             await context.SaveChangesAsync();
```
Comment (Ben0189) [inline blotztask-api/Data/Seeding/BlotzContextSeed.cs:63]:
```suggestion
     Console.WriteLine("No Label found in database. TIPS: "you might need to run database migrations first" ")  
```

File: blotztask-api/Data/Seeding/BlotzContextSeed.cs [hunk 2]
```diff
@@ -59,28 +59,15 @@ public static async Task SeedBlotzContextAsync(UserManager<User> userManager, Ro
...
             Console.WriteLine("Label creation Success.");
```
Comment (Ben0189) [inline blotztask-api/Data/Seeding/BlotzContextSeed.cs:64]:
```suggestion
```

**Review by Ben0189 — CHANGES_REQUESTED** [review]

File: blotztask-api/Data/Seeding/BlotzContextSeed.cs [hunk 3]
```diff
@@ -78,20 +78,7 @@ private static async Task SeedLabelsAsync(BlotzTaskDbContext context)
     {
```
Comment (Ben0189) [inline blotztask-api/Data/Seeding/BlotzContextSeed.cs:78]: i dont think we need this anymore

File: blotztask-api/Data/Seeding/BlotzContextSeed.cs [hunk 4]
```diff
@@ -112,10 +99,10 @@ private static async Task SeedTasksForTodayAsync(BlotzTaskDbContext context, Use
             return;
         }
 
-        var labelUrgent = await context.Labels.FirstOrDefaultAsync(l => l.Name == "Urgent");
-        var labelCompleted = await context.Labels.FirstOrDefaultAsync(l => l.Name == "Completed");
+        var labelWork = await context.Labels.FirstOrDefaultAsync(l => l.Name == "Work");
```
Comment (Ben0189) [inline blotztask-api/Data/Seeding/BlotzContextSeed.cs:102]: We can try to use [Enum](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/enum) instead

File: blotztask-api/Data/Seeding/BlotzContextSeed.cs [hunk 5]
```diff
@@ -112,10 +99,10 @@ private static async Task SeedTasksForTodayAsync(BlotzTaskDbContext context, Use
...
+        var labelPersonal = await context.Labels.FirstOrDefaultAsync(l => l.Name == "Personal");
 
-        if (labelUrgent == null || labelCompleted == null)
+        if (labelWork == null || labelPersonal == null)
         {
             Console.WriteLine("Labels are missing. Cannot seed tasks.");
```
Comment (Ben0189) [inline blotztask-api/Data/Seeding/BlotzContextSeed.cs:107]:
```suggestion
            Console.WriteLine("Label missing. Try to run migrations to seed the label. Failed to seed task");
```

**Review by Ben0189 — APPROVED** [review]

File: blotztask-api/Data/Seeding/BlotzContextSeed.cs (hunk 4 above)
Comment (PeterOu8) [reply blotztask-api/Data/Seeding/BlotzContextSeed.cs:87]: I feel like enum itself cannot identify a label and in the future there would be some additional customised labels. I am not  sure about how to use this.

Comment (Ben0189) [reply blotztask-api/Data/Seeding/BlotzContextSeed.cs:87]:
```suggestion
        var labelWork = await context.Labels.FirstOrDefaultAsync(l => l.Name == LabelType.Work.ToString());
```

**Review by Ben0189 — CHANGES_REQUESTED** [review]

**Review by sol-wizard — APPROVED** [review]

**Pull request merged**